can/interfaces/socketcand/socketcand.py

```diff
@@ -173,9 +180,15 @@ def _recv_internal(self, timeout):
     def _tcp_send(self, msg: str):
         log.debug(f"Sending TCP Message: '{msg}'")
         self.__socket.sendall(msg.encode("ascii"))
+        if os.name != "nt":
```

Comment:
You could move the OS check into `__init__` and warn the user, if the parameter was set on windows. Set `self.__tcp_tune = False` on Windows and ignore the user input.